common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -74,7 +74,7 @@ public void run() {
             AtomicReferenceFieldUpdater.newUpdater(
                     SingleThreadEventExecutor.class, ThreadProperties.class, "threadProperties");
 
-    private final Queue<Runnable> taskQueue;
+    protected final Queue<Runnable> taskQueue;
```

Comment:
How about adding a protected final getter method with the same name instead of exposing a field?

Reply:
Child classes are defined in own packages outside `io.netty.util.concurrent`, so such getters are inaccessible by children test code - so children need helper method that only delegates to said getter. Did I read your suggestion right?

Reply:
Or for example
```
SingleThreadEventLoop {
       protected final Queue<Runnable> tailTaskQueue0() {}
}
```
```
EpollEventLoop extends SingleThreadEventLoop {
   /*package-private*/ final Queue<Runnable> tailTaskQueue() {
        return tailTaskQueue0();
    }
}
```
Is this what you are suggesting @trustin ?